Data/Base/Shaders/Common/Lighting.h

```diff
@@ -171,8 +171,11 @@ float SampleShadow(float3 shadowPosition, float2x2 randomRotation, float penumbr
 #endif
 }
 
-float CalculateShadowTerm(ezMaterialData matData, float3 lightVector, float distanceToLight, uint type,
-  uint shadowDataOffset, float noise, float2x2 randomRotation, out float subsurfaceShadow, out float3 debugColor)
+// Enable to only sample the last cascade for directional lights (which will always enclose the whole shadow range)
+// #define SHADOW_FORCE_LAST_CASCADE
```

Comment:
Could instead use if constexpr, then the code always gets checked for syntax errors.

Reply:
There is no if constexpr in shader code